### PR TITLE
Profile screen opened from SpeedGrader comments has no close nor send buttons

### PR DESCRIPTION
**Pull request by idevainstr**
refs: MBL-15946
affects: Teacher
release note: Fixed profile screen top buttons in SpeedGrader comments.
test plan: see ticket

<table>
<tr><th>Before</th><th>After</th></tr>
<tr>
<td></td>
<td></td>
</tr>
</table>

### CODE REVIEW

**Comment by inst-danger** [comment]
<details><summary>Student.ipa</summary>
  <a href="https://app.bitrise.io/app/88a3641e29e4745c/build/011485a4-36e0-48d9-8450-a9ec21474dde/artifact/9991fc92c9b9cec1/p/2dcb539b6d615c0e63f5da4819f49523">
    
  </a>
</details>
<details><summary>Teacher.ipa</summary>
  <a href="https://app.bitrise.io/app/88a3641e29e4745c/build/011485a4-36e0-48d9-8450-a9ec21474dde/artifact/81312ccdde274260/p/383d89a9bebafb99c97b42a9e17a5573">
    
  </a>
</details>
<details><summary>Parent.ipa</summary>
  <a href="https://app.bitrise.io/app/88a3641e29e4745c/build/011485a4-36e0-48d9-8450-a9ec21474dde/artifact/4f1524f3dfd1ea7d/p/fb9f064ccba9b2d2c2e549ca21fd9455">
    
  </a>
</details>

**Comment by inst-danger** [comment]
Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 91.43% | 91.46% | -0.03%

[MBL-15946](https://instructure.atlassian.net/browse/MBL-15946)

#### Affected Apps: Teacher

#### Release Note: 
Fixed profile screen navigation bar items in SpeedGrader comments.
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Core/Core/People/ContextCard/Course/ContextCardView.swift [hunk 1]
```diff
@@ -38,7 +38,8 @@ public struct ContextCardView: View {
     @ViewBuilder var emailButton: some View {
         if model.permissions.first?.sendMessages == true, model.isViewingAnotherUser {
             Button(action: { model.openNewMessageComposer(controller: controller.value) }, label: {
-                Image.emailLine.foregroundColor(.textLightest)
```
[the remark this answers is not in the document]
Comment (idevainstr) [reply Core/Core/People/ContextCard/Course/ContextCardView.swift:41]: I tried that first, but it didn't set because of the view wrapper probably.

**Review by vargaat — CHANGES_REQUESTED** [review]
For some reason the mail button on my account is still invisible, running on an iPhone SE with iOS 15.1.

**Review by vargaat — APPROVED** [review]
QA+1

**Review by balintbartok — APPROVED** [review]
QA + 1

**Pull request merged**